### PR TITLE
New Folder Icons for places and mimetypes for Music Notation 

### PR DESCRIPTION
**Pull request by rolodoom**
### places

I submitted the next new icons to the scalable directory, and the equivalent for the 16x16-dark, 16x16-light and symbolic directories. 

- **AppImage**
folder-appimage.svg
Scalable, 16x16-dark, 16x16-light, symbolic
folder-appimage  folder-appimage  folder-appimage folder-appimage-symbolic

- Home Studio / Music Recording / Mixer  Folder Icon
folder-mixer.svg
Scalable, 16x16-dark, 16x16-light, symbolic
folder-mixer folder-mixer folder-mixer folder-mixer-symbolic

- Virtualbox
folder-virtualbox.svg
Scalable, 16x16-dark, 16x16-light, symbolic
folder-virtualbox folder-virtualbox folder-virtualbox folder-virtualbox-symbolic

- Windows
folder-windows.svg
Scalable, 16x16-dark, 16x16-light, symbolic
folder-windows folder-windows folder-windows folder-windows-symbolic

- Wine
folder-wine.svg
Scalable, 16x16-dark, 16x16-light, symbolic
folder-wine folder-wine folder-wine folder-wine-symbolic

- Dropbox
folder-dropbox.svg
Scalable only
folder-dropbox

- HTML files
folder-html.svg
Scalable only
folder-html

-------------------------

### mimetypes

I submitted the next new icons for 6 file types used on music notation / composition software MuseScore and Finale

**Music XML File Types**
- **MXL**
application-vnd.recordare.musicxml3.svg
application-vnd recordare musicxml3

- **MUSICXML**
application-vnd.recordare.musicxml3+xml.svg
application-vnd recordare musicxml3+xml

**MuseScore File Types**
- **MSCZ**
application-x-musescore3.svg
application-x-musescore3

- **MSCX**
application-x-musescore3+xml.svg
application-x-musescore3+xml

**Finale Notation Software File Types**
- **MUS**
application-x-wine-extension-mus.svg
application-x-wine-extension-mus

- **MUSX**
application-x-wine-extension-musx.svg
application-x-wine-extension-musx

### CODE REVIEW

**Comment by rolodoom** [comment]
New Icons for Music Notation Software MuseScore and Finale.

- MuseScore MSCZ File
application-x-musescore3 svg
- MusScore MSCX File
application-x-musescore3+xml svg
- Finale MUS File
application-x-wine-extension-mus svg
- Finale MUSX File
application-x-wine-extension-musx svg

**Comment by keeferrourke** [comment]
@rolodoom looks like you've changes some symlinks here, as well deleted some files.

Can you clean up this PR to only include the new artwork?

Also please [clean and tidy](https://github.com/keeferrourke/la-capitaine-icon-theme/wiki/Technical-Notes#automating-svg-clean-up) your files.

**Comment by rolodoom** [comment]
Hello @keeferrourke thanks for answering. I'm going to fixed everything as you answer, clean SVG and commit again to include only the new icons I made.

**Comment by rolodoom** [comment]
Hi @keeferrourke the PR is updated and files clean and tidy (hope so!)

Thanks for your patience!

File: actions/22x22 [hunk 1]
```diff
@@ -1 +1 @@
-22x22-light/
\ No newline at end of file
+22x22-dark
```
Comment (keeferrourke) [inline actions/22x22]: This shouldn't have changed

File: apps/scalable/applications-system.svg [hunk 2]
```diff
@@ -1 +0,0 @@
....2995 0 .11283.0304.21256.0911.29934.33849.46892 1.05464 1.19817 2.14845 2.18761.0956.0956.20409.1432.32555.1432.13026 0 .23877-.039.32556-.11699l1.836-1.39346c.35609.18235.74669.34297 1.17196.48201l.36455 2.3957c.009.11283.0586.20616.14976.28.0911.074.19753.1106.319.1106h2.89066c.25203 0 .40818-.12146.46907-.36455.11267-.43407.23845-1.24117.37733-2.42175.40802-.13025.8031-.29534 1.1849-.49496l1.79702 1.40641c.12146.0779.23446.11699.3385.11699.19083 0 .59885-.30797 1.22374-.9244.62522-.61626 1.05482-1.08517 1.28895-1.40641.0781-.0868.11731-.18651.11731-.29934s-.0433-.22167-.13025-.32555c-.6249-.76394-1.08934-1.36278-1.39331-1.79686.17356-.32123.34297-.71615.5079-1.1849l2.40881-.36439c.10404-.0175.19114-.074.2605-.16957.0693-.0956.10389-.19546.10389-.2995v-2.89049c.00016-.11284-.0344-.21465-.10405-.3059zm-7.53886 4.08195c-.65094.65111-1.43662.97666-2.35686.97666-.92025 0-1.70576-.32555-2.35686-.97666-.65095-.65094-.97651-1.43646-.97651-2.3567s.32539-1.70575.97651-2.35686c.6511-.65094 1.43678-.9765 2.35686-.9765.92024 0 1.70592.32556 2.35686.9765.65095.65111.9765 1.43662.9765 2.35686s-.32555 1.70576-.9765 2.3567z" fill="#fff" fill-opacity=".851852" stroke-width="1.317422"/></g></svg>
```
Comment (keeferrourke) [inline apps/scalable/applications-system.svg]: Don't delete this

File: mimetypes/scalable/application-vnd.recordare.musicxml3+xml.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<svg height="64" viewBox="0 0 64 64" width="64" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (keeferrourke) [inline mimetypes/scalable/application-vnd.recordare.musicxml3+xml.svg]: Looks good!

File: places/symbolic/folder-dropbox-symbolic.svg [hunk 4]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (keeferrourke) [inline places/symbolic/folder-dropbox-symbolic.svg]: This icon needs to be cleaned

File: places/symbolic/folder-html-symbolic.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+/home/rolo/public_html/06-GiT/la-capitaine-icon-theme/places/symbolic/folder-remote-symbolic.svg
```
Comment (keeferrourke) [inline places/symbolic/folder-html-symbolic.svg]: This is definitely not right

**Comment by keeferrourke** [comment]
@rolodoom sorry for the late response -- just reviewed everything :) Only a few more changes needed.

**Pull request merged**